app/frontend/src/service/events.ts

```diff
@@ -18,3 +25,55 @@ export async function listCommunityEvents(
   const res = await client.communities.listEvents(req);
   return res.toObject();
 }
+
+export async function getEvent(eventId: number) {
+  const req = new GetEventReq();
+  req.setEventId(eventId);
+  const res = await client.events.getEvent(req);
+  return res.toObject();
+}
+
+interface ListEventUsersInput {
+  eventId: number;
+  pageSize?: number;
+}
+
+export async function listEventOrganisers({
+  eventId,
+  pageSize,
+}: ListEventUsersInput) {
```

Comment:
You haven't included page token with these rpc calls so pagination won't work

Reply:
Good catch, building this with a broken API has really made these obviously broken things harder to spot! 🙈